src/cow-react/modules/wallet/api/components/WalletModal/options/WalletConnectOption.tsx

```diff
@@ -13,12 +16,18 @@ const BASE_PROPS = {
 }
 
 export function WalletConnectOption({ tryActivation }: { tryActivation: (connector: Connector) => void }) {
```

Comment:
Just thoughts, no need to address them now:
Looks like many `[WalletName]Option` components have the same boilerplate, can we get rid of the code duplications?
Smth like:
```
<WalletOption getWallet={getIsZengoWallet}></WalletOption>
<WalletOption getWallet={getIsAmbireWallet}></WalletOption>
...
```